hw/dv/tools/dvsim/common_sim_cfg.hjson

```diff
@@ -2,6 +2,9 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 {
+  // This is a configuration file for a simulation
+  dvsim_flow:       "sim"
```

Comment:
This is replicated already on line 11 (other files as well). IMO `flow` is sufficient. It is very unlikely an unrelated HJson will also have the same key/valid value AND for someone to pass it as input to DVSim.

Reply:
That makes sense. Let's do that. I've moved those two lines up to the top, since they sort of give the "mode" of the file (which also matches the other files that define "flow")